src/br/com/sunty/models/Category/Category.java

```diff
@@ -0,0 +1,97 @@
+package br.com.sunty.models.Category;
```

Comment:
A convenção de pacotes no Java é sempre usar tudo minúsculo, sem nem camelCase.
```suggestion
package br.com.sunty.models.category;
```